### PR TITLE
Fix/add tests for compile time default value evaluation

### PR DESCRIPTION
**Pull request by dylan-chong**

### CODE REVIEW

File: lib/absinthe/blueprint/schema.ex [hunk 1]
```diff
@@ -169,7 +169,6 @@ defmodule Absinthe.Blueprint.Schema do
   end
 
   defp build_types([:close | rest], [%Schema.InputValueDefinition{} = arg, field | stack], buff) do
-    arg = Map.update!(arg, :default_value, fn val -> {:unquote, [], [val]} end)
```
Comment (dylan-chong) [inline lib/absinthe/blueprint/schema.ex:172]: Seemed to not work, and i'm guess it was supposed to do the evaluation. Clearly it didn't

File: test/absinthe/type/object_test.exs [hunk 2]
```diff
@@ -129,4 +129,19 @@ defmodule Absinthe.Type.ObjectTest do
...
+    |> Enum.each(fn %{
+                      test_label: test_label,
+                      expected_value: expected_value
+                    } ->
+      test "for #{test_label} (evaluates default_value to '#{expected_value}')" do
+        type = Object.TestSchemaFieldsAndArgsDescription.__absinthe_type__(:field_default_value)
+
+        assert type.fields[unquote(test_label)].default_value == unquote(expected_value)
+      end
+    end)
```
Comment (dylan-chong) [inline test/absinthe/type/object_test.exs:145]: the input object's field's default_value is always nil here. Not sure if intentional or not. But i have removed this test

**Comment by dylan-chong** [comment]
@benwilson512 Would this be able to make it into 1.6? I found this issue testing 1.6.0-rc-0 on my company's codebase

File: test/absinthe/type/query_test.exs [hunk 3]
```diff
@@ -1,68 +1,51 @@
 defmodule Absinthe.Type.QueryTest do
   use Absinthe.Case, async: true
 
-  defmodule TestSchema do
```
Comment (dylan-chong) [inline test/absinthe/type/query_test.exs:4]: moved to fixtures

**Comment by binaryseed** [comment]
Thanks for this!

**Pull request merged**